File: java-misinformation/secure8.java
import org.springframework.ai.chat.client.ChatClient;
import javax.ws.rs.POST;
import javax.ws.rs.Path;
import javax.ws.rs.QueryParam;
import javax.ws.rs.core.Response;
import java.util.Map;

@Path("/ask")
public class AskResource {
    private static final String AI_WARNING = "This response is AI-generated and may contain inaccuracies.";
    private final ChatClient chatClient;

    public AskResource(ChatClient.Builder builder) {
        this.chatClient = builder.build();
    }

    @POST
    public Response ask(@QueryParam("question") String question) {
        String answer = chatClient.prompt().user(question).call().content();
        String safeResponse = AI_WARNING + "\n\n" + answer;
        return Response.ok(Map.of("answer", safeResponse)).build();
    }
}

File: csharp-misinformation/secure8.cs
using Anthropic.SDK;
using Anthropic.SDK.Messaging;

var builder = WebApplication.CreateBuilder(args);
var app = builder.Build();

const string AiWarning = "This response is AI-generated and may contain inaccuracies.";
var client = new AnthropicClient("API_KEY");

app.MapPost("/ask", async (string question) =>
{
    var response = await client.Messages.GetClaudeMessageAsync(new MessageParameters
    {
        Model = AnthropicModels.Claude3Opus,
        MaxTokens = 4096,
        Messages = [new Message { Role = RoleType.User, Content = question }]
    });
    string answer = response.Content[0].Text;
    return Results.Ok(new { answer = $"{AiWarning}\n\n{answer}" });
});

app.Run();
